addons/crm/tests/test_crm_lead_merge.py

```diff
@@ -294,3 +390,120 @@ def test_merge_method_dependencies(self):
         # 2many accessors updated
         self.assertEqual(master_lead.activity_ids, activity)
         self.assertEqual(master_lead.calendar_event_ids, calendar_event)
+
+    @users('user_sales_manager')
+    @mute_logger('odoo.models.unlink')
+    def test_merge_method_followers(self):
+        """ Test that the followers of the leads are added in the destination lead.
+
+        They should be added if:
+        - The related partner was active on the lead (posted a message in the last 7 days)
+        - The related partner is not already following the destination lead
+        """
+        self.env['crm.lead'].browse(self.lead_w_partner_company.ids).write({'type': 'opportunity'})
+        self.leads.mapped('message_follower_ids').unlink()
+        self.leads.mapped('message_ids').unlink()
+
+        (
+            follower_contact_1_lead_1,
+            follower_contact_1_lead_2,
+            follower_contact_2_lead_2,
+            follower_contact_2_lead_3,
+            follower_contact_3_lead_2,
+        ) = self.env['mail.followers'].sudo().create([{
```

Comment:
Would prefer using a standard message_subscribe, to avoid low-level manipulation of followers.